types/options.go

```diff
@@ -114,11 +120,53 @@ func loadDefaultStoreOptions() {
 
 // loadStoreOptions returns the default storage ops for containers
 func loadStoreOptions() (StoreOptions, error) {
-	storageConf, err := DefaultConfigFile()
+	baseConf, err := DefaultConfigFile()
 	if err != nil {
 		return defaultStoreOptions, err
 	}
-	return loadStoreOptionsFromConfFile(storageConf)
+
+	// Load the base config file
+	baseOptions, err := loadStoreOptionsFromConfFile(baseConf)
+	if err != nil {
+		return defaultStoreOptions, err
```

Comment:
Let's wrap those errors, otherwise they're hard to trace down the API, like:
```suggestion
		return defaultStoreOptions, fmt.Errorf("load storage options from config file: %w", err)
```

Reply:
Updated, thanks.